pkg/rpc/client/rpc.go

```diff
@@ -1150,33 +1150,26 @@ func (c *Client) GetNativeContractHash(name string) (util.Uint160, error) {
 // TraverseIterator returns a set of iterator values (maxItemsCount at max) for
 // the specified iterator and session. If result contains no elements, then either
 // Iterator has no elements or session was expired and terminated by the server.
-// If maxItemsCount is a negative number, then the full set of iterator values
-// will be returned using several `traverseiterator` calls if needed.
+// If maxItemsCount is negative, then config.DefaultMaxIteratorResultItems iterator
```

Comment:
Maybe "<= 0"? "give me 0 items" doesn't make much sense directly, but it's probably a bit more convenient to use as "give me the default number of elements" than -1.

Reply:
Yes, fixed.